src/components/Home.tsx

```diff
@@ -1,21 +1,70 @@
-import { Button, Heading, Link } from "@chakra-ui/react";
+import {
+  Box,
+  Button,
+  Container,
+  HStack,
+  Heading,
+  Image,
+  Link,
+  Grid,
+  GridItem,
+  Center,
+  Text,
+  Card,
+  CardBody,
+  Flex
+} from "@chakra-ui/react";
```

Comment:
You also should remove any of these imports that you aren't actually using (including the icons below which you aren't displaying anywhere).

Reply:
Updated